crosslink/internal/common.go

```diff
@@ -38,6 +41,31 @@ func identifyRootModule(rootPath string) (string, error) {
 	return modfile.ModulePath(rootModFile), nil
 }
 
+func buildUses(rootModulePath string, graph map[string]*moduleInfo, rc RunConfig) ([]string, error) {
+	var uses []string
+	for module := range graph {
+		// skip excluded
+		if _, exists := rc.ExcludedPaths[module]; exists {
+			rc.Logger.Debug("Excluded Module, ignoring use",
+				zap.String("module", module))
+			continue
+		}
+
+		localPath, err := filepath.Rel(rootModulePath, module)
+		if err != nil {
+			return nil, fmt.Errorf("failed to retrieve relative path: %w", err)
+		}
+		if localPath == "." || localPath == ".." {
+			localPath += "/"
+		} else if !strings.HasPrefix(localPath, "..") {
+			localPath = "./" + localPath
+		}
```

Comment:
This logic looks very similar to the logic found in `insertReplace` [here](https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/653d34f660a4223b39ffdf37f4dc81dd419887c9/crosslink/internal/crosslink.go#L75). Can we lift this out and move it into `common.go`?

Reply:
I agree. I will to it tomorrow 😉

Reply:
Addressed 27832a32b9403873e246ba99c9022783138b65de